### DISCUSSION TITLE
Levelling Guide for Alts

### DISCUSSION THREAD

**Discussion by neomanyouth** [Lailloken/Exile-UI#364] [Q&A] [answered]
Is it possible to have separate levelling guides for different characters? It seems it can only have one levelling guide at a time without overwriting.

**Comment by Lailloken** [comment, marked as answer]
Not officially, but the guide and its progress is saved in ` ini\leveling guide.ini`, so you could make a backup-copy of that, then reset the guide progress and import a new one. When you want to go back to the first character, close the tool, restore the old file, and start the script again. Off the top of my head, this should work without a problem.

**Reply by neomanyouth** [reply]
Damn you reply fast, thanks for the tip. Might be an idea for an enhancement in future to have the guide loaded be based on the character name under Active Character.
Just wanted to say I really like this mod, really great work!

**Reply by Lailloken** [reply]
Yeah, I'll look into some form of guide management. Thanks for the suggestion.

**Reply by neomanyouth** [reply]
Sounds good. Also can confirm your workaround worked perfectly.

**Comment by Lailloken** [comment]
[v1.52.6](https://github.com/Lailloken/Lailloken-UI/releases/tag/v1.52.6) now includes two additional guide "slots".
`EDIT: the second and third slot use "leveling guide2.ini" and "leveling guide3.ini", so you can rename your secondary file accordingly`
